cmd/go-mkopensource/main.go

```diff
@@ -430,9 +436,56 @@ func Main(args *CLIArgs) error {
 		}
 	}
 
+	isDirty, err := isGoModDirty()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "WARNING: could not verify if go.mod or go.sum are dirty: %s.\n", err.Error())
+	}
+
+	if isDirty {
+		fmt.Fprintln(os.Stderr, "WARNING: go.mod or go.sum are dirty.\nMake sure that these files are commited with the "+
```

Comment:
Should the workflow fail if this happens? Same for if `isGoModDirty` errs.

Reply:
Good catch! I've updated it.

Reply:
For the `err != nil` it's just a warning because we can't be sure the files were modified.